packages/react/package.json

```
@@ -6,7 +6,8 @@
     "govuk-frontend": "^3.8.0",
     "react": "^16.13.1",
     "react-dom": "^16.13.1",
-    "react-scripts": "^3.4.3"
+    "react-scripts": "^3.4.3",
+    "styled-components": "^5.2.0"
```

We don't need this package anymore

```suggestion
    "react-scripts": "^3.4.3"
```